src/components/inputs/date-input/date-input.js

```diff
@@ -169,21 +168,22 @@ const DateInput = props => {
                       clearSelection();
                     }
                   },
-                  onFocus: openMenu,
-                  onClick: openMenu,
+                  // we only do this for readOnly because the input
+                  // doesn't ignore these events, unlike when its disabled
+                  onFocus: props.isReadOnly ? undefined : openMenu,
```

Comment:
Nit: `!props.isReadonly && openMenu` if null works.

Reply:
Yeah I did that at first, but it sets to `false` when `isReadOnly` which is not acceptable.